package.json

```diff
@@ -29,25 +32,30 @@
     "lint": "biome check .",
     "format": "biome format --write ."
   },
+  "dependencies": {
+    "json-schema": "^0.4.0"
+  },
   "devDependencies": {
+    "@ai-sdk/openai": "^1.1.14",
     "@biomejs/biome": "^1.5.3",
     "@types/bun": "^1.2.4",
     "@types/json-schema": "^7.0.15",
     "@types/node": "^22.13.5",
-    "@ai-sdk/openai": "^1.1.14",
-    "openai": "^4.85.4",
+    "@typescript/native-preview": "^7.0.0-dev.20250623.1",
+    "ai": "^4.1.46",
     "dotenv": "^16.3.1",
     "husky": "^9.1.7",
-    "json-schema": "^0.4.0",
     "lint-staged": "^15.2.0",
     "mkdocs": "^0.0.1",
+    "openai": "^4.85.4",
     "openapi-types": "^12.1.3",
-    "rimraf": "^6.0.1",
-    "typescript": "^5.0.0",
-    "ai": "^4.1.46"
+    "publint": "^0.3.12",
```

Comment:
use `publint`, `unplugin-unused` to lint package.json
https://tsdown.dev/reference/config-options#publint
https://tsdown.dev/reference/config-options#unused